content/en/docs/concepts/scheduling-eviction/pod-overhead.md

```diff
@@ -10,17 +10,12 @@ weight: 30
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.18" state="beta" >}}
-
+{{< feature-state for_k8s_version="v1.24" state="stable" >}}
 
 When you run a Pod on a Node, the Pod itself takes an amount of system resources. These
 resources are additional to the resources needed to run the container(s) inside the Pod.
-_Pod Overhead_ is a feature for accounting for the resources consumed by the Pod infrastructure
-on top of the container requests & limits.
-
-
-
-
+In Kubernetes, "Pod Overhead" is a way for to account for the resources consumed by the Pod
```

Comment:
We [use italics to introduce new concepts](https://kubernetes.io/docs/contribute/style/style-guide/#use-italics-to-define-or-introduce-new-terms)
```suggestion
In Kubernetes, _Pod overhead_ is a way for to account for the resources consumed by the Pod
```